tests/ocf-url_parse-leaking-relative/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
+    <head>
+        <meta charset="utf-8"/>
+        <title>Parse a potentially leaking relative URL</title>
+        <script type="text/javascript">
+            const expected_path   = 'media/imgs/monastery.jpg';
+            const parent_relative = `../../../../../${expected_path}`;
+            function test_url() {
+                try {
+                    document.getElementById('no_scripting').style.display = 'none';
+                    document.getElementById('scripting').style.display = 'block';
+
+                    const container_root_URL = new URL('../..', document.documentURI);
+                    document.getElementById('container_root_url').textContent = container_root_URL;
+
+                    document.getElementById('test_url').textContent = parent_relative;
+
+                    const image_src = new URL(parent_relative, container_root_URL);
+                    document.getElementById('image_url_parent_paths').textContent = image_src;
+
+                    document.getElementById('expected').textContent = `{Container Root URL}${expected_path}`;
```

Comment:
I'm confused: what's {Container Root URL}? I would expect that to be a JS var, but I don't see "Container Root URL" defined (and, of course, it has spaces). Does this somehow magically interpret container_root_URL?

My other comment is that I think this test would be more readable if you defined "expected" as a variable at the top of the file, but I can't tell whether "expected" genuinely depends on stuff inside this function.

Reply:
I have added comments above

Reply:
Also, the `{Container Root URL}` is to be displayed literally on the page. The problem is that the value of this thing is, sometimes, unreadably long (e.g., in Thorium) so displaying the value here makes the test unmanagable. It refers to the value with the same label a few paragraphs down.